### PR TITLE
player: make --start-time work with --rebase-start-time=no

### PR DESCRIPTION
**Pull request by tmm1**

### CODE REVIEW

File: player/loadfile.c [hunk 1]
```diff
@@ -1139,6 +1139,9 @@ static void play_current_file(struct MPContext *mpctx)
             startpos = start;
     }
     if (startpos != MP_NOPTS_VALUE) {
+        if (!opts->rebase_start_time) {
+            startpos += mpctx->demuxer->start_time;
```
Comment (wm4) [inline player/loadfile.c]: Is there any reason not to do this in rel_time_to_abs instead? (I'm not sure.)

Comment (tmm1) [inline player/loadfile.c]: Don't think `rel_time_to_abs` helps here, since `--start-time` uses MPSEEK_ABSOLUTE

Comment (wm4) [inline player/loadfile.c]: I meant adding `start_time` to the timestamps inside of `rel_time_to_abs` instead. `rel_time_to_abs` computes the start timestamp in the first place.

Not sure if that would make sense.

Comment (wm4) [inline player/loadfile.c]: Hm, didn't check the other cases, but maybe so. Merged.

**Pull request closed without merging**